### PR TITLE
Chore/web twig web server

### PR DESCRIPTION
**Pull request by literat**
Setting up a minimal Symfony web server for local development and as a preparation for demo presentation same as https://spirit-design-system-demo.netlify.app/

Based on official Docker support for Symfony: https://github.com/dunglas/symfony-docker

Please mainly check if this is a good approach to have both the webserver and package code in one repository. I think I should also update `.gitattributes` to omit Symfony app files.

### CODE REVIEW

File: packages/web-twig/composer.json [hunk 1]
```diff
@@ -34,12 +35,20 @@
     "phpunit/phpunit": "^9",
     "mockery/mockery": "^1.5",
     "lmc/coding-standard": "^3.2",
-    "symfony/yaml": "^4.4 || ^5.4 || ^6.1",
+    "symfony/yaml": "^5.4",
```
Comment (janicekt) [inline packages/web-twig/composer.json:38]: why not symfony 6 ?

File: packages/web-twig/.dockerignore [hunk 2]
```diff
@@ -0,0 +1,30 @@
+**/*.log
```
Comment (janicekt) [inline packages/web-twig/.dockerignore:1]: I guess I'd plunge the whole thing into some kind of subfolder? Not combine bundle and symfony app for examples..

File: packages/web-twig/composer.json (hunk 1 above)
Comment (literat) [reply packages/web-twig/composer.json:38]: Because of PHP 7.4.

File: apps/web-twig-demo/templates/base.html.twig [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (literat) [inline apps/web-twig-demo/templates/base.html.twig:1]: Delete this file

File: packages/web-twig/.gitattributes [hunk 4]
```diff
@@ -2,13 +2,21 @@
 
 # Exclude build/test files from archive
 /.github            export-ignore
+/app                export-ignore
```
Comment (literat) [inline packages/web-twig/.gitattributes:5]: Remove app files.

File: packages/web-twig/composer.json [hunk 5]
```diff
@@ -34,7 +34,7 @@
...
```
Comment (literat) [inline packages/web-twig/composer.json:37]: Revert these changes.

File: packages/web-twig/phpunit.xml.dist [hunk 6]
```diff
@@ -21,6 +21,11 @@
     <php>
         <!-- E_ALL = 30719 -->
         <ini name="error_reporting" value="30719"/>
+
+        <!-- ###+ symfony/framework-bundle ### -->
```
Comment (literat) [inline packages/web-twig/phpunit.xml.dist:25]: Revert these changes.

File: packages/web-twig/.dockerignore (hunk 2 above)
Comment (literat) [reply packages/web-twig/.dockerignore]: Moved to a different directory.

File: packages/web-twig/composer.json (hunk 1 above)
Comment (literat) [reply packages/web-twig/composer.json]: Moved the change back.

File: packages/web-twig/.dockerignore (hunk 2 above)
Comment (literat) [reply packages/web-twig/.dockerignore]: App skeleton was moved to another directory.
